### DISCUSSION TITLE
SoftEther VPN is also helping.

### DISCUSSION THREAD

**Discussion by AliAlmasi** [WeAreMahsaAmini/FreeInternet#57] [Ideas]
[SoftEther VPN project](https://softether.org) can also connect during this time. It's based on SSTP and you can connect to it with any kind of device (Currently, working on Windows and Android - based on my personal use)

You can add free public servers of [VPNGate (Sister project of SoftEther)](https://www.vpngate.net/en/) to android with [this open-source SSTP tool](https://github.com/kittoku/Open-SSTP-Client)
Or, you can use the [SoftEther Client Manager on Windows](https://www.vpngate.net/en/howto_windows.aspx) (Also, on Windows you can add servers directly to your VPN settings - as described [here](https://www.vpngate.net/en/howto_windows.aspx))

Plus, if you can't access the VPN servers list (vpngate.net), you can use [mirror sites](https://www.vpngate.net/en/sites.aspx); or even better, [subscribe to receive an email daily with mirror links in it](https://www.vpngate.net/en/mail.aspx).

**Comment by AmirAghaee** [comment]
Is there any documentation for running in the personal server?

**Reply by AliAlmasi** [reply]
Yes, [here you go](https://www.vpngate.net/en/join.aspx).
